devops/group_vars/all/config.yml

```diff
@@ -43,3 +43,9 @@ unicorn_config_path: "{{ app_config_path }}/unicorn.rb"
 unicorn_pid_path: "{{ app_temp_path }}/unicorn.pid"
 unicorn_stderr_path: "{{ app_logs_path }}/unicorn.err.log"
 unicorn_stdout_path: "{{ app_logs_path }}/unicorn.log"
+
+puma_bind_path: "unix://{{ app_temp_path }}/sockets/puma.sock"
+puma_config_path: "{{ app_config_path }}/puma.rb"
+puma_pidfile_path: "{{ app_temp_path }}/pids/puma.pid"
+puma_stderr_redirect_path: "{{ app_logs_path }}/puma.err.log"
+puma_stdout_redirect_path: "{{ app_logs_path }}/puma.log"
```

Comment:
@RDrop everything under `devops` is only used for `railsbox.io` deployment (which uses unicorn), so it's better to revert this